tensorboard/util/imghdr.py

```diff
@@ -0,0 +1,175 @@
+"""Recognize image file formats based on their first few bytes."""
```

Comment:
May be add a line of reference to the original library, i believe it's
https://github.com/python/cpython/blob/3.12/Lib/imghdr.py?